### PR TITLE
Update 4.6.0 release tags 

### PR DESCRIPTION
**Pull request by santhosh-tg**

### CODE REVIEW

**Comment by amorphous-1** [comment]
Hello @santhosh-tg,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://924.qa.docs.sunbird.org/4.6.0/ .<br/> **Note:** Only _4.6.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://924.qa.docs.sunbird.org/4.6.0/developer-docs/server-installation/build-and-deploy/ <br />  <br/> **Modified sections are -** <br />

**Comment by amorphous-1** [comment]
Hello @santhosh-tg,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://924.qa.docs.sunbird.org/4.6.0/ .<br/> **Note:** Only _4.6.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://924.qa.docs.sunbird.org/4.6.0/developer-docs/server-installation/build-and-deploy/ <br />  <br/> **Modified sections are -** <br />

File: developer-docs/server-installation/build-and-deploy.md [hunk 1]
```diff
@@ -71,53 +73,53 @@ az storage blob upload-batch --destination sunbird_content_azure_storage_contain
...
+|Build/Core/APIManager|release-4.4.1_RC2|<https://github.com/project-sunbird/sunbird-devops.git>|The API Gateway|
 |Build/Core/ApiManagerEcho|master-RC1|<https://github.com/project-sunbird/sunbird-echo-service.git>|Simply echos the request|
-|Build/Core/Assessment|release-4.3.0_RC9|<https://github.com/project-sunbird/knowledge-platform.git>|Handles course assessments|
+|Build/Core/Assessment|release-4.6.0_RC2|<https://github.com/project-sunbird/knowledge-platform.git>|Handles course assessments|
 |Build/Core/Bot|release-3.9.0_RC1|<https://github.com/project-sunbird/sunbird-bot.git>|Frontend for the chatbot|
-|Build/Core/Cassandra|release-4.3.0_RC3|<https://github.com/project-sunbird/sunbird-utils.git>|Handles the Cassandra database schema|
-|Build/Core/CassandraTrigger|release-3.8.0_RC2|<https://github.com/project-sunbird/sunbird-utils.git>|Creates few Cassandra Triggers|
+|Build/Core/Cassandra|release-4.6.0_RC1|<https://github.com/project-sunbird/sunbird-utils.git>|Handles the Cassandra database schema|
+|Build/Core/CassandraTrigger|release-4.5.0_RC4|<https://github.com/project-sunbird/sunbird-utils.git>|Creates few Cassandra Triggers|
```
Comment (keshavprasadms) [inline developer-docs/server-installation/build-and-deploy.md:83]: Please check this tag

File: developer-docs/server-installation/build-and-deploy.md [hunk 2]
```diff
@@ -174,83 +177,84 @@ az storage blob upload-batch --destination sunbird_content_azure_storage_contain
...
+|Deploy/KnowledgePlatform/Learning|release-4.5.0_RC12|<https://github.com/project-sunbird/sunbird-learning-platform.git>|Deploys the Learning service on tomcat|
+|Deploy/KnowledgePlatform/Neo4j|release-4.5.0_RC12|<https://github.com/project-sunbird/sunbird-learning-platform.git>|Deploys the Neo4j plugins|
+|Deploy/KnowledgePlatform/Neo4jDefinitionUpdate|release-4.5.0_RC12|<https://github.com/project-sunbird/sunbird-learning-platform.git>|Creates / Updates the Neo4j database|
+|Deploy/KnowledgePlatform/Yarn|release-4.5.0_RC12|<https://github.com/project-sunbird/sunbird-learning-platform.git>|Deploys the Samza jobs|
+|Deploy/KnowledgePlatform/FlinkJobs|branch_or_tag: release-4.6.0_RC2, job_names_to_deploy: Select All|<https://github.com/project-sunbird/sunbird-learning-platform.git>|Deploys the Flink jobs in Kubernetes cluster|
+|Deploy/KnowledgePlatform/LoggingFileBeatsVM|branch_or_tag: release-4.5.0_RC12, hosts: Select All, tags: default|<https://github.com/project-sunbird/sunbird-devops.git>|Deploys filebeat in all the selected VMs|
```
Comment (keshavprasadms) [inline developer-docs/server-installation/build-and-deploy.md:196]: This tag is incorrect

**Comment by amorphous-1** [comment]
Hello @santhosh-tg,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://924.qa.docs.sunbird.org/4.6.0/ .<br/> **Note:** Only _4.6.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://924.qa.docs.sunbird.org/4.6.0/developer-docs/server-installation/build-and-deploy/ <br />  <br/> **Modified sections are -** <br />

**Comment by amorphous-1** [comment]
Hello @santhosh-tg,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://924.qa.docs.sunbird.org/4.6.0/ .<br/> **Note:** Only _4.6.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://924.qa.docs.sunbird.org/4.6.0/developer-docs/server-installation/build-and-deploy/ <br />  <br/> **Modified sections are -** <br />

**Review by keshavprasadms — APPROVED** [review]

**Pull request merged**